backend/lib/utils.js

```diff
@@ -17,8 +19,37 @@ const bool = (env) => env == "true";
 
 const getCookieToken = (req) => req.cookies.token;
 
-const emailRegEx = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
-const isValidEmail = (email) => emailRegEx.test(email);
+// email validation code from front-end 'src\utils\validators'
+const validateTopLevelDomain = (string) => {
```

Comment:
Maybe change the name to have the prefix `isValid`, to match the other validators

```suggestion
const isValidTopLevelDomain = (string) => {
```